packages/terra-dynamic-grid/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "terra-dynamic-grid",
+  "main": "lib/DynamicGrid.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-dynamic-grid",
+    "DynamicGrid",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-toolkit": "^1.0.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^1.0.0",
+    "terra-mixins": "^1.0.0"
+  },
+  "dependencies": {
+    "aphrodite": "^1.2.3",
```

Comment:
Each CSS-In-JS approach is quite unique and aphrodite nicely fits the requirements.  Here is the reasoning:

* styled-components - Styles are required to be bound to react element. This makes it difficult/impossible to create multiple response layouts to power a single grid
* styles-jsx - Doesn't allow for dynamic styles
* Aphrodite - Styles are decoupled from component. They are almost the same as CSS modules in that you generate the stylesheet and pass reference by key to class names. This makes it easy to generate all the responsive styles and properly pass them down to the regions.